### PR TITLE
Zer 157 front end localization

### PR DESCRIPTION
**Pull request by kromch**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @kromch 

ZER-157-FrontEnd-Localization
Translate button is named not correctly when user is not loged-in

ToDo

Patch on issue

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -21,4 +26,15 @@ def message_class_by_name(type)
...
+  def change_locale!
+    sl = I18n.locale.to_s
+    res = LN.each_with_index do |h, ind|
+      if sl == h.keys.first.to_s
+        i = user_signed_in? ? (ind + 1) % LN.length : ind - 1
+        break LN[i]
+      end
+    end
+    res == LN ? LN[0] : res
+  end
```
Comment (opazyniuk) [inline app/helpers/application_helper.rb:39]: please, rewrite this to be more readable and obvious

File: app/views/layouts/_navigation.html.slim [hunk 2]
```diff
@@ -15,6 +15,5 @@ body
       .donate-btn
         = link_to t('.donate'), "https://zerowastelviv.org.ua/pidtrymaty/", target: '_blank'
       - if current_locale?(:en)
-        = link_to t(".local_lang"), locale_path(:id => :uk), method: :patch, class: "btn btn-outline-success btn-sm mn-vd"
```
Comment (opazyniuk) [inline app/views/layouts/_navigation.html.slim:18]: ` = link_to "Українська", locale_path(:id => :uk), method: :patch, class: "btn btn-outline-success btn-sm mn-vd"`

File: app/views/layouts/_navigation.html.slim [hunk 3]
```diff
@@ -15,6 +15,5 @@ body
...
-      - else
-        = link_to t(".local_lang"), locale_path(:id => :en), method: :patch, class: "btn btn-outline-success btn-sm mn-vd"
```
Comment (opazyniuk) [inline app/views/layouts/_navigation.html.slim:20]: `= link_to "English", locale_path(:id => :en), method: :patch, class: "btn btn-outline-success btn-sm mn-vd"`

**Comment by loqimean** [comment]
Already completed by #276

**Pull request closed without merging**